client/src/partials/patient_invoice/registry/search.modal.html

```diff
@@ -39,12 +39,30 @@
             <i class="fa fa-eraser"></i> <span translate>FORM.BUTTONS.CLEAR</span>
           </a>
         </span>
-        <input type="text" class="form-control" name="reference"ng-model="ModalCtrl.params.reference">
+        <input type="text" class="form-control" name="reference" ng-model="ModalCtrl.params.reference">
         <div class="help-block" ng-messages="ModalForm.reference.$error" ng-show="ModalForm.$submitted">
           <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
         </div>
       </div>
 
+      <!-- client -->
+      <div class="form-group">
```

Comment:
I know from experience that this ui-select will crash some of the computers at Vanga ... listing every debtor in a dropdown is _far_ too big a task.   I propose you remove this `ui-select` altogether.